Guides/Development.md

```diff
@@ -76,12 +78,14 @@ Our documentation site is automatically generated from the source code using [Ja
 If you'd like to preview how new documentation you've written will look when published, you can regenerate it by running `./etc/generate-docs.sh` and then inspecting the generated HTML files in `/docs`.
 
 ## Linting and Style
-We use [SwiftLint](https://github.com/realm/SwiftLint#using-homebrew) for linting. You can see our configuration in the `.swiftlint.yml` file in the project's root directory.  Run `swiftlint` in the root directory to lint all of our files. Running `swiftlint autocorrect` will correct some types of violations.
+We use [SwiftLint](https://github.com/realm/SwiftLint#using-homebrew) for linting. You can see our configuration in the `.swiftlint.yml` file in the project's root directory.  Run `swiftlint` in the root directory to lint all of our files. Running `swiftlint --fix` will correct some types of violations.
 
 We use [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) for formatting the code. You can see our configuration in the `.swiftformat` file in the project's root directory. Our linter config contains a superset of the rules that our formatter does, so some manual tweaking may be necessary to satisfy both once the formatter is run (e.g. line length enforcement). Most of the time, the formatter should put the code into a format that passes the linter. You can run the formatter on all of the files by running `swiftformat .` from the root directory.
 
 To pass all the formatting stages of our testing matrix, both `swiftlint --strict` and `swiftformat --lint .` must finish successfully.
 
+`make lint-and-format` runs `swiftlint --fix`, `swiftlint --strict` and `swiftformat --lint .`.
```

Comment:
The `let` issues I was having in my previous [PR](https://github.com/mongodb/mongo-swift-driver/pull/715) came up again when I ran this. It seems like theres a parser warning thats generated, I could not figure out how to suppress it.

`Skipping correcting file because it produced Swift parser diagnostics: /Users/nellie.spektor/swift/driver/Sources/MongoSwift/MongoCollection+FindAndModify.swift
Correcting 'MongoNamespace.swift' (144/165)
Correcting 'Sources/MongoSwift/CursorCommon.swift' (145/165)
{
  "diagnostics" : [
    {
      "key.column" : 8,
      "key.description" : "consecutive statements on a line must be separated by ';'",
      "key.diagnostic_stage" : "source.diagnostic.stage.swift.parse",
      "key.filepath" : "\/Users\/nellie.spektor\/swift\/driver\/Sources\/MongoSwift\/MongoCollection+FindAndModify.swift",
      "key.fixits" : [
        {
          "key.length" : 0,
          "key.offset" : 7,
          "key.sourcetext" : ";"
        }
      ],
      "key.line" : 1,
      "key.severity" : "source.diagnostic.severity.error"
    },
    {
      "key.column" : 13,
      "key.description" : "keyword 'let' does not need to be escaped in argument list",
      "key.diagnostic_stage" : "source.diagnostic.stage.swift.parse",
      "key.filepath" : "\/Users\/nellie.spektor\/swift\/driver\/Sources\/MongoSwift\/MongoCollection+FindAndModify.swift",
      "key.fixits" : [
        {
          "key.length" : 1,
          "key.offset" : 10545,
          "key.sourcetext" : ""
        },
        {
          "key.length" : 1,
          "key.offset" : 10549,
          "key.sourcetext" : ""
        }
      ],
      "key.line" : 214,
      "key.severity" : "source.diagnostic.severity.warning"
    },
    {
      "key.column" : 13,
      "key.description" : "keyword 'let' does not need to be escaped in argument list",
      "key.diagnostic_stage" : "source.diagnostic.stage.swift.parse",
      "key.filepath" : "\/Users\/nellie.spektor\/swift\/driver\/Sources\/MongoSwift\/MongoCollection+FindAndModify.swift",
      "key.fixits" : [
        {
          "key.length" : 1,
          "key.offset" : 13100,
          "key.sourcetext" : ""
        },
        {
          "key.length" : 1,
          "key.offset" : 13104,
          "key.sourcetext" : ""
        }
      ],
      "key.line" : 282,
      "key.severity" : "source.diagnostic.severity.warning"
    },
    {
      "key.column" : 13,
      "key.description" : "keyword 'let' does not need to be escaped in argument list",
      "key.diagnostic_stage" : "source.diagnostic.stage.swift.parse",
      "key.filepath" : "\/Users\/nellie.spektor\/swift\/driver\/Sources\/MongoSwift\/MongoCollection+FindAndModify.swift",
      "key.fixits" : [
        {
          "key.length" : 1,
          "key.offset" : 16145,
          "key.sourcetext" : ""
        },
        {
          "key.length" : 1,
          "key.offset" : 16149,
          "key.sourcetext" : ""
        }
      ],
      "key.line" : 361,
      "key.severity" : "source.diagnostic.severity.warning"
    }
  ]
}`

Reply:
hmmm, I don't see that output when I run these myself. what versions of swiftlint and swiftformat are you using?

Reply:
![Screen Shot 2022-01-06 at 4 46 15 PM](https://user-images.githubusercontent.com/9171876/148456557-18d49e2c-6024-4b5d-aab6-89a95966c1c9.png)

Reply:
ok I get that too now that I updated swiftlint. off the top of my head I don't know what the best way to deal with this is. can you file a ticket about it?

Reply:
sure